src/components/Button/Button.scss

```diff
@@ -0,0 +1,751 @@
+  &.tiny {
+    min-width: 4.4rem;
+    min-height: 2.4rem;
+
+    span {
+      font-size: 1rem;
+    }
+  }
+
+  &.big {
+    min-width: 5.4rem;
+    min-height: 4.5rem;
+
+    span {
+      font-size: 1.3rem;
+    }
+  }
+
+  &.primary {
+    &.filled {
+      background-color: $main-theme;
+      border-radius: 0.25rem;
+      border: solid 0.2rem $main-theme;
+      color: #ffffff;
+
+      &.inverse {
+        background-color: $main-theme-lighter;
+        border-color: $main-theme-lighter;
+        color: $gray-dark;
+
+        &:focus,
+        &.focus {
+          background-color: $main-theme-lighter;
+        }
+
+        &:hover,
+        &.hover {
+          background-color: #dff4ff;
+        }
+
+        &:active,
+        &.active {
+          background-color: $main-theme-lighter;
+        }
+
+        &.disabled {
+          background-color: $gray-dark;
+          border-radius: 0.25rem;
+          border: solid 0.2rem $gray-dark;
+          color: #3e3e3e;
+          pointer-events: none;
+        }
+      }
+
+      &:focus,
+      &.focus {
+        background-color: $main-theme;
+      }
+
+      &:hover,
+      &.hover {
+        background-color: $main-theme-dark;
+      }
+
+      &:active,
+      &.active {
+        background-color: $main-theme-darker;
+      }
+
+      &.disabled {
+        background-color: $gray-light;
+        border-radius: 0.25rem;
+        border: solid 0.2rem $gray-light;
+        color: #ffffff;
+        pointer-events: none;
+      }
+
+      &.blue {
+        background-color: $primary-blue;
+        border: solid 0.2rem $primary-blue;
+      }
+
+      &.brown {
+        background-color: $primary-brown;
+        border: solid 0.2rem $primary-brown;
+      }
+
+      &.cyan {
+        background-color: $primary-cyan;
+        border: solid 0.2rem $primary-cyan;
+      }
+
+      &.green {
+        background-color: $primary-green;
+        border: solid 0.2rem $primary-green;
+      }
+
+      &.indigo {
+        background-color: $primary-indigo;
+        border: solid 0.2rem $primary-indigo;
+      }
+
+      &.orange {
+        background-color: $primary-orange;
+        border: solid 0.2rem $primary-orange;
+      }
+
+      &.pink {
+        background-color: $primary-pink;
+        border: solid 0.2rem $primary-pink;
+      }
+
+      &.purple {
+        background-color: $primary-purple;
+        border: solid 0.2rem $primary-purple;
+      }
+
+      &.slate {
+        background-color: $primary-slate;
+        border: solid 0.2rem $primary-slate;
+      }
+
+      &.teal {
+        background-color: $primary-teal;
+        border: solid 0.2rem $primary-teal;
+      }
+    }
+
+    &.outline {
+      background-color: transparent;
+      border-radius: 0.25rem;
+      border: solid 0.2rem $main-theme;
+      color: $main-theme;
+
+      &:focus,
+      &.focus {
+        background-color: transparent;
+      }
+
+      &:hover,
+      &.hover {
+        background-color: #ebebeb;
```

Comment:
As discussed in the design file, this may need to be a follow on ticket. The "fill" of the outline button in the hover state (as well as in other buttons, but it easier to demonstrate here) should be an opacity layer which interacts with the background. If we ever want to put an outline button, for example, on a light blue background, the hover state should show that hue through the button to help make the visuals more flexible and cohesive.

So instead of this:
<img width="120" alt="Screenshot 2023-08-08 at 11 59 05 AM" src="https://github.com/CDCgov/cdc-react/assets/70541324/4c1c9d6d-c10c-4fe6-a1a2-ab2f5450dbd4">
It should look more like this:
<img width="116" alt="Screenshot 2023-08-08 at 12 00 14 PM" src="https://github.com/CDCgov/cdc-react/assets/70541324/34c8b40d-b0cc-4413-a1df-b39bd59f00ed">

Reply:
Updated